### PR TITLE
Display hosting accounts list

### PR DESCRIPTION
**Pull request by tmartinfr**

### CODE REVIEW

File: compta/views.py [hunk 1]
```diff
@@ -268,3 +272,55 @@ def subscription_edit(request, subscription_id):
...
     success_url = reverse_lazy('subscription_list')
+
+
+class HostingListView(generic.View):
+    """
+    Fetch a Salt Pillar file over the GitHub API, then display it as an HTML
+    table.
+    """
+
+    def fetch_pillar(self, url, token):
```
Comment (SwannCroiset) [inline compta/views.py]: can be a static method

File: compta/views.py [hunk 2]
```diff
@@ -268,3 +272,55 @@ def subscription_edit(request, subscription_id):
...
+        headers = {'Accept': 'application/vnd.github.v3+json',
+                   'Authorization': 'token {}'.format(token)}
+        res = requests.get(url, headers=headers, timeout=5)
+        res.raise_for_status()
+        return base64.decodestring(res.json()['content'])
+
+    def get_accounts(self, pillar):
```
Comment (SwannCroiset) [inline compta/views.py]: can be a static method

File: compta/views.py [hunk 3]
```diff
@@ -268,3 +272,55 @@ def subscription_edit(request, subscription_id):
...
+        pillar_dict = yaml.load(pillar)
+        return pillar_dict['http']['users']
+
+    def filter_active(self, accounts):
```
Comment (SwannCroiset) [inline compta/views.py]: can be a static method

File: compta/views.py [hunk 4]
```diff
@@ -268,3 +272,55 @@ def subscription_edit(request, subscription_id):
...
+                    if root_path_data.get('is_active', True)
+            }
+            return account_data
+
+        accounts = {account: filter_root_path(account_data)
+                    for account, account_data in accounts.iteritems()
+                    if account_data.get('is_active', True)}
+        return accounts
+
+    def sort_accounts(self, accounts):
```
Comment (SwannCroiset) [inline compta/views.py]: can be a static method

File: compta/views.py (hunk 1 above)
Comment (tmartinfr) [reply compta/views.py]: @SwannCroiset Yeah, totally.

**Pull request merged**